api-2.2/src/main/java/org/openmrs/module/fhir2/api/impl/FhirConditionServiceImpl_2_2.java

```diff
@@ -56,22 +46,15 @@ public class FhirConditionServiceImpl_2_2 extends BaseFhirService<Condition, org
 	private SearchQuery<org.openmrs.Condition, Condition, FhirConditionDao<org.openmrs.Condition>, ConditionTranslator<org.openmrs.Condition>, SearchQueryInclude<Condition>> searchQuery;
 	
 	@Override
-	public IBundleProvider searchConditions(ReferenceAndListParam patientParam, TokenAndListParam code,
-	        TokenAndListParam clinicalStatus, DateRangeParam onsetDate, QuantityAndListParam onsetAge,
-	        DateRangeParam recordedDate, TokenAndListParam id, DateRangeParam lastUpdated, @Sort SortSpec sort,
-	        HashSet<Include> includes) {
+	public IBundleProvider searchConditions(ConditionSearchParams conditionSearchParams) {
+		if (conditionSearchParams.getClinicalStatus() == null
```

Comment:
added this check to ensure that the clinical status is provided. This is because we have two different implementations of the Condition Resource. The first translation is based on the Obs Resource and the second translation is the direct translation of the Condition Resource that was introduced in `core 2.2.0`